src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -0,0 +1,597 @@
+package com.ncc.aif;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.io.CharSource;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFFormat;
+import org.topbraid.jenax.statistics.ExecStatistics;
+import org.topbraid.jenax.statistics.ExecStatisticsListener;
+import org.topbraid.jenax.statistics.ExecStatisticsManager;
+import org.topbraid.shacl.vocabulary.SH;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Spec;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.concurrent.Callable;
+
+/**
+ * A command-line AIF validator.  For details, see <a href="https://github.com/NextCenturyCorporation/AIDA-Interchange-Format">the AIF README</a>
+ * section entitled, <i>The AIF Validator</i>.
+ */
+@CommandLine.Command(name = "validateAIF",
+        sortOptions = false,
+        synopsisHeading = "%nUsage: ",
+        descriptionHeading = "%nDescription: ",
+        optionListHeading = "%nOptions:%n",
+        description = "Used to validate Turtle files with extension .ttl",
+        mixinStandardHelpOptions = true,
+        version = "1.1.0")
+public class ValidateAIFCli implements Callable<Integer> {
```

Comment:
_Putting this here since it refers to an unmodified file._

I think the README.md file should be updated to bring the usage message in sync with program output:
- Add -V / --version and -t [num]
- Update -h / --help
- Could you add backtick in --pm?  My bug.

Also, it occurs to me that the beginning of the README only indirectly mentions the validator CLI.  That first bullet should refer to the appropriate section in the README ("The AIF Validator").

Reply:
Fixed

Reply:
Could you fix the SNAPSHOT number in the README?
And in the first bullet:

> This ontology can be validated using the SHACL constraints file in      `src/main/resources/com/ncc/aif/aida_ontology.shacl`.

how about adding something like, " by the AIF Validator."

Reply:
We're still on 1.1.0, right? That's what's in the POM.

Reply:
Oh for goodness sakes, every now and then I mix the AIF and TopBraid versions.  Yes, the README is correct.

I agree that they don't have to use the validator to use the SHACL file, but I disagree that the current wording is sufficient, in that I believe the validator CLI should be mentioned in the intro.  When those bullets were written, there might not even have been a validator class written yet (or if there was, it was rudimentary).  Now it's a full-featured command-line tool (with programmatic access) for validating AIF, which we recommend users of AIF use to validate their KBs.  It's a core "resource to support [AIF]," which is what the opening paragraph and bullets are describing.  I'm open to other ways of mentioning it, perhaps in a separate bullet?

Reply:
Added bullet to README concerning what this repository contains. Let me know if it's sufficient

Reply:
Looks good-- thanks.  Resolving.